aws/client/aws-client-rulesengine/src/test/java/software/amazon/smithy/java/aws/client/rulesengine/RunnerTest.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.java.aws.client.rulesengine;
+
+public class RunnerTest {
+    // TODO
```

Comment:
Intentional?

Reply:
I need to add JMESPath extraction before I can actually fill this in and pass all the tests. I'll have a JMESPath extraction followup PR.